### PR TITLE
Added appendTo and prependTo with tests see #641

### PR DESCRIPTION
**Pull request by digihaven**

### CODE REVIEW

File: lib/api/manipulation.js [hunk 1]
```diff
@@ -102,6 +102,26 @@ var uniqueSplice = function(array, spliceIdx, spliceCount, newElems, parent) {
   return array.splice.apply(array, spliceArgs);
 };
 
+exports.appendTo = function(target) {
+	if (typeof target === 'string') {
```
Comment (jugglinmike) [inline lib/api/manipulation.js]: This project uses two-space soft tabs. Could you change the code style here to match?

File: test/api/manipulation.js [hunk 2]
```diff
@@ -496,6 +496,24 @@ describe('$(...)', function() {
     });
   });
 
+  describe('.appendTo', function() {
+
+    it('(Node) : should add element as last child', function() {
```
Comment (jugglinmike) [inline test/api/manipulation.js]: The test description is incorrect. It describes a Node-accepting signature, but the test invokes `appendTo` with a Cheerio instance (which is elsewhere described as `"$(...)"`)

File: test/api/manipulation.js [hunk 3]
```diff
@@ -496,6 +496,24 @@ describe('$(...)', function() {
...
+      $('<li class="plum">Plum</li>').appendTo($fruits);
+      expect($fruits.children().eq(3).hasClass('plum')).to.be.ok();
+    });
+
+  });
+
+  describe('.prependTo', function() {
+
+    it('(html) : should add element as first child', function() {
```
Comment (jugglinmike) [inline test/api/manipulation.js]: The test description is incorrect. It describes an HTML-accepting signature, but the test invokes `prependTo` with a Cheerio instance (which is elsewhere described as `"$(...)"`)

**Comment by jugglinmike** [comment]
Thanks for the patch, @digihaven! Could you add some more tests? Some important ones that come to mind:

- Invoking with a string of HTML
- Invoking with a "node" instance
- Invoking with a selector
- Invoking with an Array

**Comment by fb55** [comment]
Fixed with #797

**Pull request closed without merging**